CMakeLists.txt

```diff
@@ -72,7 +72,10 @@ boost_${CLP_FFI_JS_BOOST_ARCHIVE_VERSION_PART}.tar.gz"
     URL_MD5 "53aeccc3167909ee770e34469f8dd592"
 )
 message(STATUS "Fetching Boost.")
+# Temporarily enable verbose output for boost download progress
+set(FETCHCONTENT_QUIET OFF)
```

Comment:
Since someone who's having download issues would need this for any library we download using `FetchContent`, I feel like we should just disable `FETCHCONTENT_QUIET` entirely rather than only for Boost.

Reply:
Yeah, perhaps just after `include(FetchContent)`.